ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -4392,6 +4390,81 @@ def test_torch_instance_index_select(
     )
 
 
+@st.composite
+def _get_clamp_inputs(draw):
+    shape = draw(
+        helpers.get_shape(
+            min_num_dims=1, max_num_dims=5, min_dim_size=2, max_dim_size=10
+        )
+    )
+    x_dtype, x = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("numeric"),
+            shape=shape,
+        )
+    )
+    min = draw(st.booleans())
+    if min:
+        max = draw(st.booleans())
+        min = draw(
+            helpers.array_values(
+                dtype=x_dtype[0], shape=shape, min_value=-25, max_value=0
```

Comment:
I found out the issue, it's because the dtype generation includes "unsigned integer", so during data generation, negative values will be like "inverted" or something, so there are cases where minimum turns out to be larger than maximum, which leads to hypothesis raising an error.

try limiting all ranges from 0 - max.

you can use something like `min=0, max=25` for generating min, and `min=26, max=50` for generating max

hope this helps!

Reply:
Yes this has helped! Thanks!